views/viewGallery.php

```diff
@@ -12,14 +12,21 @@
             require('layout/commentForm.php');
         }
     }
-    if ($currentPage != 1) {
-        echo '<a href="index.php?page='.($currentPage - 1).'"><<</a>';
-    }
-    for ($i = 1; $i <= $pagesTotal; $i++) {
-        echo '<a href="/index.php?page='.$i.'">'.$i.'</a>';
-    }
-    if ($currentPage != $pagesTotal) {
-        echo '<a href="index.php?page='.($currentPage + 1).'">>></a>';
+    if ($pagesTotal > 1) {
+        if ($currentPage != 1) {
+            echo '<a href="index.php?page='.($currentPage - 1).'"><<</a>';
+        }
+        for ($i = 1; $i <= $pagesTotal; $i++) {
+            if ($i == $currentPage){
+                echo $i;
+            }
+            else {
+                echo '<a href="/index.php?page='.$i.'">'.$i.'</a>';
+            }
+        }
+        if ($currentPage != $pagesTotal) {
+            echo '<a href="index.php?page='.($currentPage + 1).'">>></a>';
+        }
     }
 ?>
 <h3>That's it ! There is no more pictures !</h3>
```

Comment:
```suggestion
<h3>That's it ! There is no more pictures !</h3>
```
A afficher uniquement sur la dernière page

Reply:
bien vu